.github/CONTRIBUTING.md

```diff
@@ -13,6 +18,7 @@ submit a pull request. Please, try to follow these guidelines when you do so.
 * Open an issue with a descriptive title and a summary in grammatically correct, complete sentences.

```

Comment:
I don't find "people won't read it" as a compelling argument to keep off-putting guidelines.